src/main.rs

```diff
@@ -76,6 +83,8 @@ static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
 
 const GLOBAL_CSS: &str = include_str!("../res/app.css");
 const ABOUT_IMAGE: &[u8] = include_bytes!("../res/about.png");
+#[cfg(any(target_os = "macos", target_os = "linux"))]
+const TRAY_ICON: &[u8] = include_bytes!("../res/linux/space-acres.png");
```

Comment:
Later I'll probably make a copy of this icon in `unix` subdirectory rather than `linux` and make it much smaller. There is no need to have 256x256 image as a tray icon.